mantidproject/mantidimaging
# Load flat and dark, before and after

Fixes #477 

Test:
- Open mantidimaging
- Load Data that has before and after for flat and dark
- Open operations
- Select flat fielding operation
- Test out the data and ensure it "looks right"
